src/main/java/command/AddTaskCommand.java

```diff
@@ -0,0 +1,35 @@
+package command;
+
+import dukeexception.DukeException;
+import store.Storage;
+import store.TaskList;
+import userinteraction.Ui;
+
+/**
+ * Abstract command class for adding tasks.
+ */
+abstract class AddTaskCommand extends Command {
+    /**
+     * Public constructor.
+     *
+     * @param inputArr String array from user input.
+     */
+    public AddTaskCommand(String[] inputArr) {
+        super(inputArr);
+    }
+
+    @Override
+    public void execute(TaskList taskList, Ui ui, Storage storage) throws DukeException {
+
+    }
```

Comment:
I notice that there is nothing inside the method, are you forgot to write it?

Reply:
This is an abstract class so there is nothing meant to be in it, will leave a comment inside the method to specify. Thanks for noting it out!